content/blog/automated-releases-part-two.md

```diff
@@ -0,0 +1,234 @@
+
+```yaml
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: 'Testing'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: 
+          - '1.14'
+          - '1.13'
+          - '1.12' 
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v1.1.2
+        with:
+          go-version: '${{ matrix.go }}'
+      - name: Invoking go test
+        run: go test ./...
+  
+  release:
+    name: 'Release to GitHub'
+    runs-on: ubuntu-latest
+    needs: 
+      - test
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v1.1.2
+        with:
+          go-version: '1.14'
+      - name: Invoking go vet and binaries generation
+        run: |
+          go vet ./...
+          GOOS=darwin GOARCH=amd64 go build -o=.github/workflows/asyncapi-parser.darwin.amd64 ./cmd/api-parser/main.go
+          GOOS=linux GOARCH=amd64 go build -o=.github/workflows/asyncapi-parser.linux.amd64 ./cmd/api-parser/main.go
+          GOOS=windows GOARCH=amd64 go build -o=.github/workflows/asyncapi-parser.windows.amd64.exe ./cmd/api-parser/main.go
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 13
+      - name: Add plugin for conventional commits
+        run: npm install conventional-changelog-conventionalcommits
+        working-directory: ./.github/workflows
+      - name: Release to GitHub
+        working-directory: ./.github/workflows
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          GIT_AUTHOR_NAME: asyncapi-bot
+          GIT_AUTHOR_EMAIL: info@asyncapi.io
+          GIT_COMMITTER_NAME: asyncapi-bot
+          GIT_COMMITTER_EMAIL: info@asyncapi.io
+        run: npx semantic-release
+```
+
+You see, you can still have your project pure from any JavaScript specific files and references. Everything you need to run release with the JavaScript community tooling is only in the release related configuration.
+
+## Conclusion
+
+I don't think I can ever understand this "hate" towards JavaScript. I think, though, that you can "hate" the language, but if you see some amazing tooling built with it, that can increase your productivity, grit your teeth, put bias aside, and enjoy life. Especially if in exchange you get this excellent feature, notification about release under the Issue and Pull Request:
+
+![pr info about release](/images/posts/pr-indicator.png)
+
+If you think this blog post is not clear and is missing some more explanation, you most probably did not read the first part of the automation story. I highly recommend you to read [Full automation of release to NPM and Docker Hub with GitHub Actions and Conventional Commits](/blog/automated-releases/). I also recommend to [join our Slack](https://www.asyncapi.com/slack-invite/) for further discussion.
```

Comment:
> If you think this blog post is not clear and is missing some more explanation, you most probably did not read the first part of the automation story.

This is very presumptuous. I'd rephrase it so it doesn't sound like "if you don't understand, it's not my fault, it's yours because you didn't read my other article" 😄

Reply:
I pushed a change, have a look